### DISCUSSION TITLE
Doubt Regarding Deep Source: Java

### DISCUSSION THREAD

**Discussion by UG-SEP** [TesseractCoding/NeoAlgo#4038] [Q&A] [answered]
#4036 
My pull request is showing Deep Source: Java Error 
Well What is this and how to fix it can to tell me sir .
Please

**Comment by HarshCasper** [comment, marked as answer]
Hi @UG-SEP 

We have ripped off the DeepSource Java Config. Apologies for this experience! You will not face any issues from now.

**Reply by UG-SEP** [reply]
thanks
